securedrop_client/gui/widgets.py

```diff
@@ -19,38 +19,107 @@
 import logging
 import arrow
 import html
-from PyQt5.QtCore import Qt, pyqtSlot
-from PyQt5.QtGui import QIcon, QPalette, QBrush, QColor, QFont
+from PyQt5.QtCore import Qt, pyqtSlot, QSize, QTimer
+from PyQt5.QtGui import QIcon, QPalette, QBrush, QColor, QFont, QLinearGradient
 from PyQt5.QtWidgets import QListWidget, QLabel, QWidget, QListWidgetItem, QHBoxLayout, \
     QPushButton, QVBoxLayout, QLineEdit, QScrollArea, QDialog, QAction, QMenu, QMessageBox, \
-    QToolButton, QSizePolicy, QTextEdit, QStatusBar
+    QToolButton, QSizePolicy, QTextEdit, QStatusBar, QGraphicsDropShadowEffect
 from typing import List
 from uuid import uuid4
 
 from securedrop_client.db import Source, Message, File, Reply
 from securedrop_client.logic import Client
-from securedrop_client.resources import load_svg, load_image
+from securedrop_client.resources import load_svg, load_icon, load_image, load_icon_button
 from securedrop_client.utils import humanize_filesize
 
 logger = logging.getLogger(__name__)
 
 
-class StatusBar(QStatusBar):
+class TopPane(QWidget):
+    """
+    Top pane of the app window.
+    """
+
     def __init__(self):
         super().__init__()
-
         self.setStyleSheet('''
-            QStatusBar { background-color: #fff; }
             QStatusBar::item { border: none; }
-            QPushButton { border: none; }
+            QPushButton { border: none; color: #fff }
+            QStatusBar#activity_status_bar { color: #fff; }
+            QWidget#error_bar { background-color: #f22b5d }
```

Comment:
Note: I got the values of the colors using Gimp's color-sampling tool on images from here: https://projects.invisionapp.com/share/XDRB535NR9Z, which contains wireframes for the error status bar, etc (click through the story). I'm grabbing these colors because they are more up-to-date in invisionapp than they are in zeplin. So this is just to get it to look more like the design we're leaning towards. Eventually we'll code in a better way to store and use a color scheme.